docs/minio-admin-complete-guide.md

```diff
@@ -362,10 +383,18 @@ Restarted `play` successfully.
 
 ```
 NAME:
-  mc admin info - get MinIO server information
+  mc admin info - display MinIO server information
 
 FLAGS:
-  --help, -h                       show help
+  --config-dir value, -C value  path to configuration folder
+  --quiet, -q                   disable progress bar display
+  --no-color                    disable color theme
+  --json                        enable JSON lines formatted output
+  --debug                       enable debug output
+  --insecure                    disable SSL certificate verification
+  --limit-upload value          limits uploads to a maximum rate in KiB/s, MiB/s, GiB/s. (default: unlimited)
+  --limit-download value        limits downloads to a maximum rate in KiB/s, MiB/s, GiB/s. (default: unlimited)
```

Comment:
these are not in docs on purpose, they simply did not want to pollute with the same content. Since its well understood these are global flags.

Reply:
The global flags is displayed for each command, which is redundant.